src/System.Net.Http/src/System/Net/Http/Unix/HttpClientHandler.Unix.cs

```diff
@@ -0,0 +1,145 @@
+        }
+
+        public CookieContainer CookieContainer
+        {
+            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+        }
+
+        public ClientCertificateOption ClientCertificateOptions
+        {
+            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+        }
+
+        public DecompressionMethods AutomaticDecompression
+        {
+            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+        }
+
+        public bool UseProxy
+        {
+            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+        }
+
+        public IWebProxy Proxy
+        {
+            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+        }
+
+        public bool PreAuthenticate
+        {
+            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+        }
+
+        public bool UseDefaultCredentials
+        {
+            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+        }
+
+        public ICredentials Credentials
+        {
+            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+        }
+
+        public bool AllowAutoRedirect
+        {
+            get
+            {
+                return _curlHandler.AutomaticRedirection;
+            }
+            set
+            {
+                _curlHandler.AutomaticRedirection = value;
+            }
+        }
+
+        public int MaxAutomaticRedirections
+        {
+            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+        }
+
+        public long MaxRequestContentBufferSize
+        {
+            get { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+            set { throw NotImplemented.ByDesignWithMessage("HTTP stack not implemented"); }
+        }
+
+        #endregion Properties
+
+        #region De/Constructors
+
+        public HttpClientHandler()
```

Comment:
It is symmetric with WinHttpHandler. Also if in future, we want to support CurlHandler as a separate binary (like we do for WinHttpHandler), it will be easy to do that